### PR TITLE
{devel}[GCCcore/11.2.0] Qt5 v5.15.2

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso
CORE_CNT=16

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
node2606.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/532aac76b725911f3ef186641a4a6f38 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 13 out of 26 (1 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.6
See https://gist.github.com/322c80414ab67538671189fa540fcf3e for a full test report.

**Comment by Micket** [comment]
Not sure, past few months have been a blur. Probably not

**Comment by SebastianAchilles** [comment]
@Micket A patch for Qt5 with GCC 11 is in https://github.com/Micket/easybuild-easyconfigs/pull/12. Building works. On my virtual machine however I still get `Sanity check failed: no file found at 'lib/libQt5WebEngine.so'`. But this could be related to a different issue.

**Comment by SebastianAchilles** [comment]
`Qt5WebEngine` depends on Python 2.

This PR depends on:
- [ ] https://github.com/Micket/easybuild-easyconfigs/pull/12
- [ ] https://github.com/easybuilders/easybuild-easyconfigs/pull/14336

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
jsfc01.int.jusuf.sebastian.cluster - Linux rocky linux 8.4, x86_64, AMD EPYC 7742 64-Core Processor (zen2), 1 x NVIDIA GRID V100-4C, 460.73.01, Python 3.6.8
See https://gist.github.com/2f1508efd6a4cb031986f31f3fa4f6ca for a full test report.

File: easybuild/easyconfigs/q/Qt5/Qt5-5.15.2-GCCcore-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    ('DBus', '1.13.18'),
+    ('libevent', '2.1.12'),
+    ('libGLU', '9.0.2'),
+    ('libjpeg-turbo', '2.0.6'),
+    ('NSS', '3.69'),
+    ('snappy', '1.1.9'),
+    ('JasPer', '2.0.33'),
+    ('bzip2', '1.0.8'),
+    ('OpenSSL', '1.1', '', True),
+]
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/q/Qt5/Qt5-5.15.2-GCCcore-11.2.0.eb:63]:
```suggestion
    ('ICU', '69.1'),
]
```

**Review by SebastianAchilles — CHANGES_REQUESTED** [review]
`ICU` is no longer included in `Qt5`:
https://doc.qt.io/qt-5/windows-requirements.html#icu

**Comment by boegel** [comment]
@boegelbot please test @ generoso
CORE_CNT=16

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3100.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/0fdef6e76f0cc6c1314c2b002fa8ef46 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2639.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/82fdd20f8c9b5842863662048ed6fde6 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3518.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/395f35d6906e7da295ca197f1101c732 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @Micket!

**Pull request merged**